app/scripts/controllers/main.js

```diff
@@ -51,27 +54,69 @@
         else if (exercisesDatabase[exerciseKey].Section === 'Lower') {
           lowerWorkout.push(exercisesDatabase[exerciseKey]);
         }
+        else if (exercisesDatabase[exerciseKey].Section === 'Whole Body') {
+          wholeBody.push(exercisesDatabase[exerciseKey]);
+        }
+        else if (exercisesDatabase[exerciseKey].Section === 'Recovery') {
+          recovery.push(exercisesDatabase[exerciseKey]);
+        }
       }
 
-      // Create a Workout
-      // Loop 3x:
-      for (var k= 0; k < 3; k++) {
-        // pick random number between 1 and coreWorkout.length
-        var randomInt = Math.floor((Math.random() * coreWorkout.length));
-
-        // splice from array
-        var randomCore = coreWorkout.splice(randomInt,1);
-        // push to todaysWorkout
+      /**
+       ** Create a Workout. 2 loops through each Section array.
+       */
+      // Loop through 2 times
+      for (var k= 0; k < 2; k++) {
+
+        // Pick Core
+        var randomIntCore = Math.floor((Math.random() * coreWorkout.length));
+        // Splice from array
+        var randomCore = coreWorkout.splice(randomIntCore,1);
+        // Push to todaysWorkout
         todaysWorkout.push(randomCore);
-        console.log(todaysWorkout);
-        // console.log(coreWorkout.length);
-        // pick random from Lower
-        // pick random from Upper
+
```

Comment:
@crossm2  I learned a formatting tip here you might find interesting. When I first wrote lines 71-98, I created blocks of code based on sections of the body. For example, Line 78 starts the code block for Lower, and this block covers all the code related to Section Lower. 

The next commit has a different format - the blocks are grouped by the function of the code. Here's a screenshot of the next commit for comparison. The code related to finding a random number is all grouped together.

![screen shot 2016-08-20 at 4 41 02 pm](https://cloud.githubusercontent.com/assets/4665383/17834429/e6f59a68-66f4-11e6-9faf-9a52a2aabd94.png)